packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -213,7 +214,7 @@ const Bulletin = ({
               // eslint-disable-next-line jsx-a11y/aria-role
               <span role="text">
                 {offScreenText && (
-                  <VisuallyHiddenText lang={lang}>
```

Comment:
We can keep this as it was, because if the language is not passed it will be `lang=null` which is the same as `const hiddenTextProps = lang ? { lang } : {};`